core/pkg/opencost/summaryallocation_json.go

```diff
@@ -35,6 +35,16 @@ func (sa *SummaryAllocation) ToResponse() *SummaryAllocationResponse {
 		return nil
 	}
 
+	// if the efficiency has already been set,
+	// prefer that since it has been calculated elsewhere
+	// and matches the sorting criteria more closely
+	efficiency := sa.Efficiency
+	if efficiency == 0 {
```

Comment:
Because the OC code now instantiates `sa.Efficiency` when `NewSummaryAllocation()` is called, just want to double check that this condition is still working for you on the KCM side of things.

Reply:
thanks Thomas - I just reviewed KCM and we use `result := new(opencost.SummaryAllocation)` as the ctor, so we are all covered!